easybuild/easyconfigs/p/PyGObject/PyGObject-3.34.0-GCCcore-8.2.0-Python-3.7.2.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'PythonPackage'
+
+name = 'PyGObject'
+version = '3.34.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pygtk.org/'
+description = """Python Bindings for GLib/GObject/GIO/GTK+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2acb0daf2b3a23a90f52066cc23d1053339fee2f5f7f4275f8baa3704ae0c543']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('GLib', '2.60.1'),
+    ('GObject-Introspection', '1.60.1', versionsuffix),
+    ('PyCairo', '1.18.0', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['include/pygobject-%(version_major)s.0/pygobject.h'],
+    'dirs': [],
```

@Flamefire Please also check for the traditional `lib/python*/site-packages`

```suggestion
    'dirs': ['lib/python%(pyshortver)s/site-packages'],

---

FYI: I didn't as I felt it is unneccessary because you do a `python -c 'import foo'` which is superior.

---

@Flamefire That just shows the `import` works, but it doesn't check *where* it's imported from...